package.json

```diff
@@ -6,7 +6,7 @@
 	"scripts": {
 		"pretest": "if exist cypress\\reports; then rmdir /S/Q cypress\\reports; fi;",
 		"generate:report": "node reporter.js",
-		"cy:gui:stable:regression_test:school_api_test:local": "cypress open --browser edge -e tags=\"@stable_test and @regression_test and @school_api_test\",environmentName=local",
+		"cy:gui:stable:regression_test:school_api_test:local": "cypress open --browser chrome -e tags=\"@stable_test and @regression_test and @school_api_test\",environmentName=local",
```

Comment:
Ah you need to change it back to the 'edge' browser, 'chrome' was only for your local.